### DISCUSSION TITLE
Home Assistant OS 13.1

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3552] [Announcements]
## Home Assistant Operating System

* Enable NTFS and exFAT drivers for all targets (#3516) @lamauny

## Home Assistant Yellow

* Add LED indication for button handling in Yellow's U-Boot (#3536) @sairon

## Home Assistant Green

* Implement device wipe using the hardware button on Green (#3523) @sairon
* Linux: Update kernel to 6.6.46 (#3529) @sairon
* Enable 1.5MBd serial console on Green for new installs (#3527) @sairon

## Open Virtual Appliance

* Fix iwlwifi driver failing to find .pnvm files for Intel 6E (AX210) (#3549) @sairon
* Linux: Update kernel to 6.6.46 (#3529) @sairon

## Generic x86-64

* Fix iwlwifi driver failing to find .pnvm files for Intel 6E (AX210) (#3549) @sairon
* Linux: Update kernel to 6.6.46 (#3529) @sairon
* Fix CPU ID for D2xxx/N2xxx in GRUB2 loader patch (#3526) @sairon

## Hardkernel ODROID

* Linux: Update kernel to 6.6.46 (#3529) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.46 (#3529) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.46 (#3529) @sairon

## Generic aarch64

* Fix iwlwifi driver failing to find .pnvm files for Intel 6E (AX210) (#3549) @sairon
* Linux: Update kernel to 6.6.46 (#3529) @sairon

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/13.1'>Home Assistant OS 13.1</a>.</em>

**Comment by Anto79-ops** [comment]
There are no issues on my NUC 12 (x86-64) with USB SkyConnect and USB Aeotec Z-Stick 7.

**Reply by havrancek** [reply]
I've lost all my ports on HA blue - zigbee is not working

**Comment by alemgarrido** [comment]
Generic X86-64, Had to reinstall 12.4 and recover a Backup. System won't boot slot. Clearly because of the kernel update.

**Reply by rogerleung0411** [reply]
Same here. it stuck in rescue mode after auto-reboot. please take a look seriously @sairon

**Comment by TroelsS** [comment]
My Rpi 3B is stuck in an endless restart cycle after updating to 13.1. Several others are reporting similar issues with Rpi 3B and Rpi 4 here:
https://community.home-assistant.io/t/haos-13-1-upgrade-unable-to-boot/762530

**Comment by HAPO64** [comment]
It seems my Rpi 4B 64GB stucks after updating from 12.4 to 13.1. 
Wlan never comes up. After power up, there is some activity seen on the led indicator.  
After the third power up reboot with long waiting periods, HAOS comes up with 13.1 and functional HA core configs. 
Is there a fault repair function in the bbot loader??

**Comment by dpgh947** [comment]
13.0 and 13.1 seem to have caused all sorts of different problems on different systems - is this being actively looked at? I have a 2gb Pi4 which is stuck on 12.4 - 13.0 or 13.1 just causes continual reboots. I see larger memory Pi4s and also generic x86 and vm systems also having various problems. It is hard to tell from forums and issues now whether this is being investigated or not.

**Reply by GonnyTech** [reply]
I'm running HAOS 13.1 on a Pi 2B I had in a closet doing nothing. Works fine, but I'm thinking about overclocking CPU and RAM. Would someone recommend it?

**Comment by jarruda** [comment]
x64 Ubuntu+VirtualBox setup here, failed to update from 12.4.   I was able to start fresh with 13.1 and recover from a full backup, so the issue appears to be in the upgrade process.  I think I still have the VM disk image that failed to boot post-upgrade if there are valuable logs there.
